libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -348,35 +340,23 @@ - (void)dealloc
 - (BOOL)loginWithCompletion:(SFOAuthFlowSuccessCallbackBlock)completionBlock
                     failure:(SFOAuthFlowFailureCallbackBlock)failureBlock
 {
-    return [self loginWithCompletion:completionBlock failure:failureBlock account:nil];
+    return [self loginWithCompletion:completionBlock failure:failureBlock credentials:[[SFUserAccountManager sharedInstance] oauthCredentials]];
```

Comment:
Yeah, this feels like we should be able to specify `credentials` of `nil`, and then `SFAuthenticationManager` would be responsible for generating the initial credentials object in that case.